src/lib.rs

```diff
@@ -257,17 +257,24 @@ impl<T: AsBytes, X> LocatedSpan<T, X> {
         &self.fragment
     }
 
-    fn get_columns_and_bytes_before(&self) -> (usize, &[u8]) {
+    fn get_unoffsetted_slice(&self) -> &[u8] {
```

Comment:
Hmm, the name doesn't make it clear what this does, but I can't think of a better one. :/
Also some comment to explain what it does would be good

Reply:
Yes, I considered `get_original_size`, as that is pretty much the intent.  But I didn't, since it has no way of reconstructing the original length.  So `unoffsetted` is what it actually does, it undos the offset.

As for the larger question about a possibly missing trailer of `get_line()`; Yes, I was a bit worried about that myself, but decided that it's not a big problem in my main use-case of reporting parse errors.  In cases I can think of that involves marking an interval (and not just a position) of a line, I think I would have two `LocatedSpan`s to combine, where each of them would (probably) be "the rest of input from a starting point".

But I agree that this should be explained somehow, both in i a comment at `get_unoffsetted_slice` and in the docstring of `get_line`.  I'll try to write something.